api/envoy/config/trace/v2/trace.proto

```diff
@@ -47,7 +47,11 @@ message Tracing {
 
       google.protobuf.Any typed_config = 3;
     }
+
+    // Whether to attach logs for streaming events to spans.
+    bool log_stream_events = 4;
```

Comment:
Can you add a release note for this? Do we need other docs as well for that this does exactly (in terms of what events are logged by default, etc.)? I have a general question as to whether this should be a config option on the HTTP connection manager vs. the driver, and gated there. I think that makes more sense? WDYT?